alloydb-jdbc-connector/src/main/java/com/google/cloud/alloydb/Connector.java

```diff
@@ -145,8 +144,8 @@ Socket connect(InstanceName instanceName) throws IOException {
         instances.computeIfAbsent(
             instanceName,
             k -> {
-              RateLimiter<Object> rateLimiter =
-                  RateLimiter.burstyBuilder(RATE_LIMIT_BURST_SIZE, RATE_LIMIT_DURATION).build();
+              DefaultRateLimiter rateLimiter =
+                  new DefaultRateLimiter(RATE_LIMIT_BURST_SIZE, RATE_LIMIT_DURATION);
```

Comment:
I might not be reading this correctly. Is this 2 permits per second with a warmup of 30s?

Ideally we want 2 initial requests, and then 1 request / 30 s to match the other connectors.